### PR TITLE
feat(pushbox): add pushbox db config and module

### PR DESCRIPTION
**Pull request by chenba**
Because:
 - we want the pushbox api module to talk directly to the pushbox
   database

This commit:
 - add pushbox db connection config and module to auth-server

closes #fxa-5766

### CODE REVIEW

**Comment by chenba** [comment]
This PR follows https://github.com/mozilla/fxa/pull/13981.  Please review that PR first.

File: packages/fxa-auth-server/lib/pushbox/index.js [hunk 1]
```diff
@@ -74,6 +75,13 @@ module.exports = function (log, config, statsd) {
     };
   }
 
+  // eslint-disable-next-line
```
Comment (chenba) [inline packages/fxa-auth-server/lib/pushbox/index.js:78]: This isn't used yet.  The next PR in this epic will.

**Review by bbangert — APPROVED** [review]
LGTM!

**Comment by jackyzy823** [comment]
Dear developers, i have some questions about pushbox. Will `mozilla-services/pushbox` finally be replaced by direct pushbox db calls in fxa-auth-server?

As a selfhosting user, i used  `mozilla-services/pushbox` previously and it create db schema already , now db-migrations create db schema too. So there's a conflict in `create index` statement because it is already exists. so can 1) "if not exists" be added to "create index" statment or 2)  drop index  before create in db-migrations for pushbox .

Thanks.

**Comment by chenba** [comment]
@jackyzy823 Thank you for the feedback.  Is it possible for you to run https://github.com/mozilla/fxa/pull/13994/files#diff-724829b6978532134587453f623120ce3b9e458dae704f2a1611cbab5646f5b4R1-R9 but with the value of the level set to '2'?  Once that exists, the create table migration won't be attempted.

**Comment by jackyzy823** [comment]
@chenba 
Well , idea is interesting but it will fail with `Patch level in metaTable (%s) is incorrect after this patch (%s)'` Ref:https://github.com/chilts/mysql-patcher/blob/557009f9bcd41b89fb237179b45cd19e2dd0e989/index.js#L333

Besides, I hope there's a workaround suitable for both situations 1) newly created database 2) existing database. 

Thanks.

**Comment by chenba** [comment]
@jackyzy823 Unfortunately even the latest version of MySQL doesn't support any sort of "IF NOT EXISTS" syntax for indexes.

**Comment by chenba** [comment]
@jackyzy823 Thanks for the reminder on using just a single create statement.  I've created a PR: https://github.com/mozilla/fxa/pull/14226.

**Pull request merged**